lib/experimental/new_embed.dart

```diff
@@ -407,6 +415,27 @@ class NewEmbed {
     await modules.start();
   }
 
+  void _notifyIfWebKit() {
+    // See https://bugs.webkit.org/show_bug.cgi?id=199866.
+    if (window.navigator.vendor.contains('Apple') &&
+        !window.navigator.userAgent.contains('CriOS') &&
+        !window.navigator.userAgent.contains('FxiOS')) {
+      dialog.showOk('Possible delay', '''
```

Comment:
nit: consider just "OK" or "I understand". "Possible delay" sounds like it's enabling a delay.

Reply:
Not sure I understand. "Possible delay" is the title text for the dialog.

Reply:
oops. I misread the arguments.

Reply:
LGTM